tests/integration/test_integration.py

```diff
@@ -809,6 +811,7 @@ def test_invalid_entry_point_verification_3rdparty(tmpdir):
     ).assert_failure()
 
 
+@pytest.mark.skipif(IS_LINUX_ARM64 or IS_MAC_ARM64, reason="No p537 wheel published for ARM yet.")
```

Comment:
In the future we can publish p537 for aarch64, and enable this test for that platform. But it would require moving it to the pantsbuild org so that it has access to the self-hosted runners, and futzing with its CI config, and I didn't want to wait on all that to get this PR going.

Reply:
It's easy enough to just use Circle CI to publish for that. I'll set that up.